packages/rollup-plugin-html/test/rollup-plugin-html.test.ts

```diff
@@ -650,10 +653,13 @@ describe('rollup-plugin-html', () => {
       'x.css',
       'y.css',
       'image-b.svg',
+      'image-d.png',
+      'video.mp4',
     ];
 
     for (const name of expectedAssets) {
       const asset = getAsset(output, name);
+      console.log(asset, name);
       expect(asset).to.exist;
       expect(asset.source).to.exist;
```

Comment:
console?

Reply:
Yeah just for debugging purposes, this test is failing in a very strange way, suddenly it cannot find the image-c asset as soon as I add image-d. And I have no idea why 😭  I tried to play around with it and change stuff around but this bug keeps happening.